texts/index.js

```diff
@@ -425,9 +425,9 @@ texts.push([
 	"Отчитывайтесь о неполадках и предлагайте код в нашем <a href=\"https://github.com/Vita3K/Vita3K\">репозитории на GitHub</a>.", // 33
 	"Что нового", // 34
 	"У нас есть блог, в котором мы регулярно публикуем подробные отчеты о текущем прогрессе в нашем деле. Заходите почитать о нововведениях!", // 35
-	"Quickstart", // 36
-	"Blog", // 37
-	"Пока нет", // 38(made by nishinji, translated by deepl)
+	"Руководство", // 36
+	"Блог", // 37
+	"Пока не готово", // 38(made by nishinji, translated by Dum4G)
```

Comment:
you can delete this comment
It's a marker that a non-native translator has translated it